backend/app/api/endpoints/deliveries.py

```
@@ -478,6 +623,272 @@ def decide_loop_item_workflow_node(
+    "/loop-items/{item_id}/workflow-plan",
+    response_model=WorkflowPlanView,
+)
+async def submit_loop_item_workflow_plan(
+    item_id: str,
+    values: WorkflowPlanSubmit,
+    automation_run_id: str = Header(
+        default="",
+        alias="X-Wegent-Automation-Run-ID",
+        include_in_schema=False,
+    ),
+    db: Session = Depends(get_db),
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)